cypress/integration/articleBodySpec.js

```diff
@@ -0,0 +1,126 @@
+import config from '../support/config/services';
+import {
+  copyrightDataWindow,
+  firstHeadlineDataWindow,
+  firstParagraphDataWindow,
+  firstSubheadlineDataWindow,
+  getElement,
+  placeholderImageLoaded,
+  renderedTitle,
+  visibleImageNoCaption,
+  visibleImageWithCaption,
+  shouldContainText,
+} from '../support/bodyTestHelper';
+
+Object.keys(config)
+  .filter(service => config[service].pageTypes.articles !== undefined)
```

Comment:
I think something like this would be more readable

```
const serviceHasArticlePageType = service => config[service].pageTypes.articles !== undefined;

Object.keys(config)
  .filter(serviceHasArticlePageType)
```

Reply:
Ah I've just requested this is changed back. I don't mind which way it's done, but it should be done consistently across integration files, here it's done in a const and in another place in an if and in another inline as it was originally. Let's pick one way and apply it everywhere

Reply:
ah no worries. I wasn't thinking about the bigger picture.

Reply:
We'll do it separately this is clearing far more tech debt than its creating :)